### PR TITLE
Only allow entering absolute path to textfields with autocomplete

### PR DESCRIPTION
**Pull request by Lucaszw**
Close #586

fixed white-space in view

fixed if statement problem

### CODE REVIEW

File: digits/views.py [hunk 1]
```diff
@@ -435,6 +436,11 @@ def path_autocomplete():
 
     """
     path = flask.request.args.get('query','')
+
+    if not os.path.isabs(path) :
+        # Only allow absolute paths by appending forward slash
+        path = '/' + path
```
Comment (lukeyeager) [inline digits/views.py]: @IsaacYangSLA can you try this on Windows? Maybe something like this would be better:
```py
path = os.path.sep + path
```

**Comment by lukeyeager** [comment]
It's a little surprising to have a slash appended to my path for me, but it's also pretty clear why it's being done.
@jmancewicz what do you think? Too astonishing? Useful?

**Comment by Lucaszw** [comment]
Close #586

**Comment by Lucaszw** [comment]
@lukeyeager I found that appending the backslash helps avoids confusion as to whether the component is active or not. Ie someone may start writing a relative path, and just assume the the textfield isn't intelligent enough to validate the path (instead of thinking that relative paths are not allowed).

File: digits/views.py (hunk 1 above)
Comment (IsaacYangSLA) [inline digits/views.py]: Sure.

File: digits/views.py [hunk 2]
```diff
@@ -435,6 +436,11 @@ def path_autocomplete():
 
     """
     path = flask.request.args.get('query','')
+
+    if not os.path.isabs(path) :
+        # Only allow absolute paths by appending forward slash
```
Comment (gheinrich) [inline digits/views.py]: technically, isn't it more correct to say "prepending"?

**Comment by Lucaszw** [comment]
I updated prepend statement and comment

**Comment by lukeyeager** [comment]
Looks good and works great! Squash and I'll merge.

**Pull request merged**